toolchain/lowering/lowering_function_context.h

```diff
@@ -45,6 +45,11 @@ class LoweringFunctionContext {
     return it->second;
   }
 
+  // This is required for handling typetype stub reference.
+  auto TryToReuseLocal(SemanticsNodeId node_id) -> bool {
+    auto it = locals_.find(node_id);
+    return it != locals_.end();
+  }
   // Returns a local (versus global) value for the given node in loaded state.
```

Comment:
Please keep in mind we prefer to have a blank line between functions for style.

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace